airbyte-integrations/connectors/source-jdbc/src/main/java/io/airbyte/integrations/source/jdbc/AbstractJdbcSource.java

```diff
@@ -335,10 +335,13 @@ public AutoCloseableIterator<JsonNode> queryTableIncremental(final JdbcDatabase
             connection -> {
               LOGGER.info("Preparing query for table: {}", tableName);
               final String quotedCursorField = sourceOperations.enquoteIdentifier(connection, cursorField);
-              final StringBuilder sql = new StringBuilder(String.format("SELECT %s FROM %s WHERE %s > ?",
+              // read source rows where the cursor field is NULL in incremental mode
+              // this can be used by TABLES ingestion where DELETED IS NULL
```

Comment:
We just need to be careful when using this- we know it's listed as 'Incremental' on Airbyte, but it behaves more like Snapshot accumulation.

eg, we are collecting multiple rows for a single non-deleted entry. We are just having some updated value, which we distinguish by airbyte_emitted_at, or normalized_at.